package.json

```diff
@@ -53,8 +55,10 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react": "^7.37.2",
+    "eslint-plugin-sql": "^3.2.1",
     "kysely-codegen": "^0.17.0",
     "kysely-ctl": "^0.10.1",
+    "pg": "^8.13.2",
```

Comment:
Is this a peer dep of the lint plugin?

Reply:
IIRC it's a peer dep of `kysely-codegen`!